src/main/java/Duke.java

```diff
@@ -1,10 +1,223 @@
+        int result = -1;
+        try {
+            result = Integer.parseInt(userInputs.split(" ")[1]);
+        } catch (Exception exception) {
+            printFormatErrorMessage(DONE_FORMAT);
+        }
+        return result;
+    }
+
+    /**
+     * Method to create a ToDo task.
+     *
+     * @param taskManager TaskManager object that handles the tasks list.
+     * @param userInputs Raw user inputs from scanner.
+     */
+    public static void addToDoTask(TaskManager taskManager, String userInputs) {
+        String payload = getUserPayload(userInputs);
+        if (isStringEmpty(payload)) {
+            printFormatErrorMessage(TODO_FORMAT);
+            return;
+        }
+        taskManager.createToDoTask(payload);
+    }
+
+    /**
+     * Method to create a Deadline Task.
+     *
+     * @param taskManager TaskManager object that handles the tasks list.
+     * @param userInputs Raw user inputs from scanner.
+     */
+    public static void addDeadlineTask(TaskManager taskManager, String userInputs) {
+        String payload = getUserPayload(userInputs);
+        if (isStringEmpty(payload)) {
+            printFormatErrorMessage(DEADLINE_FORMAT);
+            return;
+        }
+        String[] taskInfo = payload.split("/by");
+        if (taskInfo.length < 2) {
+            printFormatErrorMessage(DEADLINE_FORMAT);
+        } else if (isAnyStringEmpty(taskInfo)) {
+            printFormatErrorMessage(DEADLINE_FORMAT);
+        } else {
+            taskManager.createDeadlineTask(taskInfo[0], taskInfo[1]);
+        }
+
+    }
+
+    /**
+     * Method to create a Event task.
+     *
+     * @param taskManager TaskManager object that handles the tasks list.
+     * @param userInputs Raw user inputs from scanner.
+     */
+    public static void addEventTask(TaskManager taskManager, String userInputs) {
+        String payload = getUserPayload(userInputs);
+        if (isStringEmpty(payload)) {
+            printFormatErrorMessage(EVENT_FORMAT);
+            return;
+        }
+        String[] taskInfo = payload.split("/at");
+        if (taskInfo.length < 2) {
+            printFormatErrorMessage(EVENT_FORMAT);
+        } else if (isAnyStringEmpty(taskInfo)) {
+            printFormatErrorMessage(EVENT_FORMAT);
+        } else {
+            taskManager.createEventTask(taskInfo[0], taskInfo[1]);
+        }
+
+    }
+
+    /**
+     * Check if a string is empty. Returning true if it does not exist or is empty.
+     *
+     * @param s Given string to check.
+     * @return boolean if the string is empty.
+     */
+    public static boolean isStringEmpty(String s) {
+        return (s == null || s.isEmpty());
+    }
+
+    /**
+     * Check if any string in a given string array is empty.
+     *
+     * @param strings Given String array.
+     * @return True if any of a string in the string array is empty.
+     */
+    public static boolean isAnyStringEmpty(String[] strings) {
```

Comment:
If the code is self-explanatory, refrain from repeating the description in a comment just for the sake of 'good documentation'.